Xamarin.Essentials/FileSystem/FileSystem.ios.tvos.watchos.macos.cs

```diff
@@ -38,6 +41,34 @@ static string GetDirectory(NSSearchPathDirectory directory)
             }
             return dirs[0];
         }
+
+        static string[] PlatformGetAppPackageDirectories(string path)
+        {
+            if (string.IsNullOrEmpty(path))
+                throw new ArgumentNullException(nameof(path));
+
+            if (path.StartsWith("/"))
```

Comment:
This is not required on Android as it defaults to the root of the Asset files list. If either of you think that there's a nicer way to ensure root paths aren't used, let me know.